src/main/java/Factory/EmployeeContactFactory.java

```diff
@@ -0,0 +1,28 @@
+package Factory;
```

Comment:
/**
 * @author Daniella Burgess 219446482
 * This is EmployeeContactFactory.java
 * Date 05 April 2022
 */